### PR TITLE
Adding code to mix Pile and expert data

### PR DESCRIPTION
**Pull request by mrcabbage972**

### CODE REVIEW

**Comment by ontocord** [comment]
Looks good. Did i see we are removing 'meta' field? I thought we want to keep that?

File: Makefile [hunk 1]
```diff
@@ -1,3 +1,4 @@
 setup_dev:
 	pip install -r requirements.txt
 	pre-commit install
+	pip install -e .
```
Comment (alireza1989) [inline Makefile:4]: Using this may cause inconsistency.  Do you need to edit the code for installed packages?

**Review by alireza1989 — APPROVED** [review]
LGTM.

**Comment by mrcabbage972** [comment]
@ontocord The meta field doesn't have a consistent schema and therefore the files fail to load with HF Datasets. Are there specific metadata fields you want to keep?

**Pull request merged**